.gitignore

```diff
@@ -15,7 +15,7 @@
 # vendor/
 
 # Ignore binary
-./kam
+kam
```

Comment:
This doesn't do what you think it does.

It's matching on anything with `kam` in the name, which means that it will ignore files with kam in them.

Reply:
(I fixed this when the newly generated documentation files weren't picked up).